lib/components/popups/CourseDisplayPopup.tsx

```diff
@@ -168,46 +168,48 @@ const CourseDisplayPopup: FC = () => {
   };
 
   // Handles post add course post response
-  const handlePostAddCourse = (plan: Plan) => (data): void => {
-    let newUserCourse: UserCourse;
-    if (data.errors === undefined && courseToShow !== null) {
-      newUserCourse = { ...data.data };
-      dispatch(updateCurrentPlanCourses([...currentCourses, newUserCourse]));
-      const allYears: Year[] = [...plan.years];
-      const newYears: Year[] = [];
-      allYears.forEach(updateYears(newYears, newUserCourse));
-      const newPlan: Plan = { ...plan, years: newYears };
-      dispatch(updateSelectedPlan(newPlan));
-      const newPlanList = [...planList];
-      for (let i = 0; i < planList.length; i++) {
-        if (planList[i]._id === newPlan._id) {
-          newPlanList[i] = newPlan;
+  const handlePostAddCourse =
+    (plan: Plan) =>
+    (data): void => {
+      let newUserCourse: UserCourse;
+      if (data.errors === undefined && courseToShow !== null) {
```

Comment:
ah i'm guessing a lot of these changes are from running prettier. Thanks miseok!